mino/minogrpc/server.go

```diff
@@ -513,9 +545,11 @@ func sendSingle(s sender, msg proto.Message, addr mino.Address) error {
 
 // receiver implements mino.receiver
 type receiver struct {
-	errs chan error
-	in   chan *OverlayMsg
-	name string
+	errs    chan error
+	in      chan *OverlayMsg
+	name    string
+	stop    chan interface{}
```

Comment:
The general advice is to use `chan struct{}` when you don't care about the type.